.scripts/makePIServiceData.js

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+/**
+ * Create a JSON file to be consumed by the Product Info Service
+ *
+ * @TODO Accept a destination file path via CLI arguments
+ * @TODO Maybe convert "tags" value to an array?
+ */
+const fs = require('fs');
+const path = require('path');
```

Comment:
`path` seems to be unused and could be removed.

Reply:
Good catch